app/main.py

```diff
@@ -1,2 +1,59 @@
+from typing import Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 8) -> None:
+        self.capacity = capacity
+        self._size = 0
+        self.table = [[] for _ in range(self.capacity)]
+
+    def get_hash(self, key: Hashable) -> int:
+        return hash(key) % self.capacity
+
+    def __setitem__(self, key: Hashable, value: Hashable) -> None:
+        if self._size >= self.capacity * 2 / 3:
+            self._resize()
+
+        hash_index = self.get_hash(key)
+        slot = self.table[hash_index]
+
+        key_found = False
+        index = 0
+        while index < len(slot):
+            existing_hash, existing_key, existing_value = slot[index]
+            if existing_hash == hash_index and existing_key == key:
+                slot[index] = (hash_index, key, value)
+                key_found = True
+                break
+            index += 1
+
+        if not key_found:
+            slot.append((hash_index, key, value))
+            self._size += 1
+
+    def __getitem__(self, key: Hashable) -> Hashable:
```

Comment:
You should not specify a return type of Hashable in this context because the value associated with a key in a dictionary or hash table may not necessarily be of type `Hashable`